source/_references/searching.md

```diff
@@ -0,0 +1,213 @@
+```
+
+Example JSON response:
+```json
+{
+  "total": 126,
+  "links": {
+    "next": {
+      "href": "/sourceimages/myorganization?limit=20&offset=120"
+    },
+    "prev": {
+      "href": "/sourceimages/myorganization?limit=20&offset=80"
+    }
+  },
+  "items": [
+    // ... Image objects
+  ]
+}
+```
+
+### Deep Paging Beyond 10,000 Hits
+
+If you want to page beyond 10'000 hits, you need to use a so-called _cursor_.
+With a cursor you can only page forward by one "page" and not randomly access a page.
+
+You can use this cursor value as input for the _offset_ parameter to page to the next page.
+A link to the next page is provided in every response.
+
+If you don't submit an _offset_ number and your result has more than 10'000 hits, it will automatically
+return a link for the next page with the cursor value in it.
+
+A typical response for an organization with more than 10'000 hits looks like the following:
+
+Example JSON response:
+```json
+{
+   "total" : 74088,
+   "links" : {
+      "next" : {
+         "href" : "/sourceimages/myorganization?limit=100&offset=VX3_3FBUQW9KNDlQYXMyTmtDUHhOd2NtOWtOR0ptTTJRMllURXlZekl5"
+      }
+   },
+   "cursor" : "VX3_3FBUQW9KNDlQYXMyTmtDUHhOd2NtOWtOR0ptTTJRMllURXlZekl5",
+   "items": [
+    // ... Image objects
+  ]
+}
+```
+
+## Filtering for particular images
+
+You can filter the list by the fields returned by a sourceimage (and its user defined metadata).
+Just append the field-name and value to your list request.
+
+The following returns all images with height = 1200.
+
+```bash
+curl -H 'Content-Type: application/json' -X GET 'https://api.rokka.io/sourceimages/myorganization?height=1200'
+```
+
+The following fields are available for each image for filtering:
+ - `hash` (string): the image hash;
+ - `binaryhash` (string): the image binary-hash;
+ - `size` (integer): the image filesize, in bytes;
+ - `format` (string): the image format, as the file extension ("png", "jpg");
+ - `width` (integer): the image width, in pixels;
+ - `height` (integer): the image height, in pixels;
+ - `created` (date): the image creation date;
+
+You can also filter by fields defined in the [User Metadata](/documentation/references/usermetadata.html),
+those fields are accessible by using the `user:` and the correct [field type](user-metadata.html) prefixes.
+
+The following example will return all images where the field `my_integer_fiel` has a value of `3`:
+
+```bash
+curl -H 'Content-Type: application/json' -X GET 'https://api.rokka.io/sourceimages/myorganization?user:int:my_integer_field=3'
+```
+
+## Prefix filtering
+
+Returning all images which start with a certain value in a field is as easy as appending a star:
+
+```bash
+curl -H 'Content-Type: application/json' -X GET 'https://api.rokka.io/sourceimages/myorganization?name=plan*'
+```
+
+Returns all images having their name starting with "_plan_", such as "_plan_t", "_plan_e".
```

Comment:
underscore doesn't work somehow if you look here:
https://github.com/rokka-io/rokka-site/blob/6f601a55757f8d3111265657cf03c4fd25c729e4/source/_references/searching.md#prefix-filtering

not sure how to fix, maybe together with the `"` it doesnt work?